### DISCUSSION TITLE
Help with text height offset

### DISCUSSION THREAD

**Discussion by avalonv** [python-pillow/Pillow#6620] [Q&A] [answered]
Hii, could someone help me out with something a bit silly? I'm trying to dynamically bold text, and for that I can't use ImageDraw.multiline_text, so I'm basically implementing it from scratch. However, I can't seem to get the distance between lines to be the same as the multiline_text method, and I have no idea which method I'm supposed to use to calculate it (I've tried several).

Some images which demonstrate the issue:

with `ImageDraw.multiline_text` (this is what I want)
![image](https://user-images.githubusercontent.com/29720696/192181947-d3a9d832-df33-489b-82e9-edd6ee3fb18a.png)

compared with `ImageFont.getbbox(text, anchor='lt')`, and same thing without the anchor:
![image](https://user-images.githubusercontent.com/29720696/192184128-c3adb88f-38bb-478a-becd-82c78847902e.png)

Here's the script I'm using (relevant part @ line 70):
https://gist.github.com/avalonv/6547c8103f588542f2da959d0741db7d

I'm kinda new to this so if you have any pointers for why it isn't working I'd appreciate it a lot :D

**Comment by radarhere** [comment, marked as answer]
Replace
```python
v_offset += font.getbbox(line)[3]
```
with
```python
v_offset += font.getbbox("A")[3] + 4
```
to better mirror https://github.com/python-pillow/Pillow/blob/ef7e1ee49efd2eaf939eeef751dbbbc62aa786cc/src/PIL/ImageDraw.py#L387-L394

**Reply by avalonv** [reply]
Holy shit.... This works!!!! I spent like, I swear to god, maybe 6 hours trying to figure out how to get this to behave like multiline_text does.

Thank you sooooo much <3
